src/locale/locales/fr/messages.po

```diff
@@ -1102,17 +1048,13 @@ msgstr "Post supprimé."
 msgid "Description"
 msgstr "Description"
 
-#: src/view/screens/Settings.tsx:760
-#~ msgid "Developer Tools"
-#~ msgstr "Outils de dév"
-
 #: src/view/com/composer/Composer.tsx:218
 msgid "Did you want to say anything?"
 msgstr "Vous vouliez dire quelque chose ?"
 
 #: src/view/screens/Settings/index.tsx:504
 msgid "Dim"
-msgstr ""
+msgstr "Faible"
```

Comment:
So here, we're talking about the second toggle in this screenshot:

![image](https://github.com/bluesky-social/social-app/assets/411336/a8754067-68f2-40b5-b0cc-61593e8e8352)

Here, I think that `Dim` is meant as in a _light intensity level_, so I think that _Atténué_ is better: it's what is used on my Hue scene setting for example.

```suggestion
msgstr "Atténué"
```